pages/detention-locations.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <loading-message v-if="loading" />
+  <map-component
+    v-else
+    :pageInfo="pageInfo"
+    :originalMarkers="originalMarkers"
+    :filtersGeneralInfo="filtersGeneralInfo"
+    :filterProperties="filterProperties"
+    iconBehaviour="changing"
+    :includeTimeline="false"
+  >
+  </map-component>
+</template>
+
+<script>
+import { SPARQLQueryDispatcher } from "~/assets/js/SPARQLQueryDispatcher";
+import APIDataHandler from "~/assets/js/APIDataHandler";
+import json from "../big-query-output.json";
+import FilteringMethods from "../assets/js/FilteringMethods";
+import MapComponent from "../components/MapComponent.vue";
+import LoadingMessage from "../components/LoadingMessage.vue";
+import Swal from "sweetalert2";
+import filterDescriptions from "../public/filterDescriptions.json";
+
+definePageMeta({
+  layout: "default",
+});
+
+export default {
+  components: { MapComponent, LoadingMessage },
+  data: () => ({
+    pageInfo: {
+      title: "Places of Detention for Accused Witches",
+      html: '<div>This map shows the different locations for accused witches place of detention. These people were imprisoned either before or after their witchcraft trial and could be <b class="font-bold">imprisoned multiple times and in different locations</b>. This dataset has records for only <b class="font-bold">560</b> instances of imprisonment for the <b class="font-bold">3212</b> named witches. Again, for most of the accused witches, the surviving documentation does not provide information about their imprisonment.</div>',
+      footer: "witches.is.ed.ac.uk",
+      confirmButtonText: "Close",
+      type: "info",
+      showCloseButton: true,
+    },
+    queryOutput: json,
+    sparqlUrl: "https://query.wikidata.org/sparql",
+    wikiPages: [],
+    loading: true,
+    originalMarkers: [],
+    filtersToFind: [
+      ["socialClass", "changing"],
+      ["occupation", "changing"],
+    ],
+    filtersGeneralInfo: {
+      title: "Accused witch filters",
+      filtersShowing: true,
+    },
+    filterProperties: {
+      sex: {
+        label: "Gender",
+        description: "",
+        filters: {
+          male: {
+            label: "Male",
+            active: true,
+            iconUrl: "/images/witch-single-blue.png",
+          },
+          female: {
+            label: "Female",
+            active: true,
+            iconUrl: "/images/witch-single-orange.png",
```

Comment:
just putting this on a random line but noticed in the meetings page there is a console.log that needs removing. Thanks Ruby

Reply:
done

Reply:
thank you

Reply:
thank you